buildSrc/src/main/groovy/com/uber/okbuck/core/model/base/Scope.groovy

```diff
@@ -54,17 +56,27 @@ class Scope {
         jvmArgs = jvmArguments
         this.depCache = depCache
 
-        extractConfigurations(configurations)
+        extractConfiguration(configuration)
     }
 
     static Scope from(Project project,
-                      Set<String> configurations,
+                      String configuration,
                       Set<File> sourceDirs = ImmutableSet.of(),
                       @Nullable File resDir = null,
                       List<String> jvmArguments = ImmutableList.of(),
                       DependencyCache depCache = ProjectUtil.getDependencyCache(project)) {
-        Set<Configuration> useful = DependencyUtils.useful(project, configurations)
-        String key = useful.collect { it.name }.toSorted().join("_")
+        Configuration useful = DependencyUtils.useful(project, configuration)
+        return from(project, useful, sourceDirs, resDir, jvmArguments, depCache)
+    }
+
+    static Scope from(Project project,
+                      Configuration configuration,
+                      Set<File> sourceDirs = ImmutableSet.of(),
+                      @Nullable File resDir = null,
+                      List<String> jvmArguments = ImmutableList.of(),
+                      DependencyCache depCache = ProjectUtil.getDependencyCache(project)) {
+        Configuration useful = DependencyUtils.useful(configuration)
+        String key = useful ? useful.name : "n/a"
```

Comment:
lets change "n/a" to "None"

Reply:
changed it to `--none--` just in case someone keeps a `None` configuration